### ISSUE TITLE
Effects are not working on G5 KC

### ISSUE THREAD

**Issue by limeraiin** [rzrwolf/LED-manager-for-Clevo#1]
The program saved me from the horrible CCC app on Gigabyte G5 KC. Color changes and brightness works flawlessly but the effects are not working.

**Comment by rzrwolf** [comment]
What Clevo model is G5 based on?

**Comment by limeraiin** [comment]
NH5x_7xDCx_DDx I presume

**Comment by PRSXFENG** [comment]
is based on NH55DPQ, Control Centre 3.0, works fine for setting colour despite being 1 zone only (set with left), no animations either on stock CC 3.0 so might not work

EDIT: with that said there is 1 cycle animation on boot, this animation cant be set in cc3

**Comment by rzrwolf** [comment]
I don't have any working versions of these laptops around, so i can't test the effects or settings..

But it seems, that as much as CCC 3.0 allows - it is possible to duplicate in my software..

**Comment by Manish8531** [comment]
Hey rzwolf I am Manish8531 want to ask that can you please build a app that can control the keyboard light and also have animations in it I can assure you that our laptop hardware is capable just a software problem as no effect is availabel in the control centre of gigabyte g5 mf laptop this laptop is basically a rebranded clevo laptop 
For this I can pay you
This not only me asking you but there are hell lot of people who wants this.

Please help 
I really have great hope on you.

**Comment by Manish8531** [comment]
Hey rzwolf I am Manish8531 want to ask that can you please build a app that can control the keyboard light and also have animations in it I can assure you that our laptop hardware is capable just a software problem as no effect is availabel in the control centre of gigabyte g5 mf laptop this laptop is basically a rebranded clevo laptop 
For this I can pay you
This not only me asking you but there are hell lot of people who wants this.

Please help 
I really have great hope on you.

**Comment by Manish8531** [comment]
Hey rzwolf I am Manish8531 want to ask that can you please build a app that can control the keyboard light and also have animations in it I can assure you that our laptop hardware is capable just a software problem as no effect is availabel in the control centre of gigabyte g5 mf laptop this laptop is basically a rebranded clevo laptop 
For this I can pay you
This not only me asking you but there are hell lot of people who wants this.

Please help 
I really have great hope on you.

**Comment by Mojzesz007** [comment]
Hello
I'm using your app to control my gigabyte g5 kf (rebranded clevo). It works perfectly for that I thank you.
But my laptop doesn't have half of the options your app provides. I humbly request a branch of your project with only compatible functions for laptops such as mine.
This laptop has one zone colour keyboard (in your app it is left zone tested works), and no cover lights.
With high regards

TLDR: request remove all options but left zone control from your app and post as new project

PS: ability to set backlight to white extremely helps with readability for me i can't stress enough how much i'm grateful
